aws/resource_aws_codebuild_project.go

```diff
@@ -1490,6 +1544,13 @@ func resourceAwsCodeBuildProjectSourceHash(v interface{}) int {
 	if v, ok := m["git_clone_depth"]; ok {
 		buf.WriteString(fmt.Sprintf("%s-", strconv.Itoa(v.(int))))
 	}
+	if v, ok := m["git_submodules_config"]; ok && len(v.([]interface{})) > 0 && v.([]interface{})[0] != nil {
+		m := v.([]interface{})[0].(map[string]interface{})
+
+		if v, ok := m["fetch_submodules"]; ok {
+			buf.WriteString(fmt.Sprintf("%s-", strconv.FormatBool(v.(bool))))
```

Comment:
It looks like the change here triggered a semi-related test configuration to start failing:

```
--- FAIL: TestAccAWSCodeBuildProject_SecondarySources_CodeCommit (25.31s)
    testing.go:640: Step 0 error: Check failed: Check 3/4 error: aws_codebuild_project.test: Attribute 'secondary_sources.3525046785.source_identifier' not found
```

If you wouldn't mind, can you please fix the hash value of that test or remove the hash-dependent value check? Thanks!

Reply:
Hash value updated. Not sure how to remove the hash dependent check without removing the whole test case, there isn't much documentation (that I could find) surrounding hashes within tests.